src/comments.js

```diff
@@ -154,7 +154,12 @@ function attach(comments, ast, text) {
 
       if (followingNode) {
         // Always a leading comment.
-        addLeadingComment(followingNode, comment);
+        if (followingNode.type === "Identifier" &&
```

Comment:
I'm not sure if this is the best place to put this code, but couldn't find anywhere else.